.github/pull_request_template.md

```diff
@@ -1,12 +1,12 @@
 ## What changes were proposed in this pull request?
+(Provide a one-liner summary of the changes in the **PR Title** field above.)
```

Comment:
```suggestion

Provide a one-liner summary of the changes in the **PR Title** field above.  It should be in the form of `HDDS-1234. Short summary of the change`.
```

Reply:
Thanks for the suggestions @adoroszlai. I made the changes accordingly.